develop/schemadoc/PackageManifest/Manifest.md

```diff
@@ -11,9 +11,10 @@ Manifest file of a package.
 |Name|Occurrences|Description|
 |--- |--- |--- |
 |***Sequence***|||
-|&nbsp;&nbsp;[Name](xref:Manifest.Name)||Specifies the name of the package.|
-|&nbsp;&nbsp;[Version](xref:Manifest.Version)||Specifies the version of the package.|
-|&nbsp;&nbsp;[Content](xref:Manifest.Content)||Specifies the content of the package.|
-|&nbsp;&nbsp;[DeleteFiles](xref:Manifest.DeleteFiles)|[0, 1]|Specifies files to delete.|
-|&nbsp;&nbsp;[DeploymentOptions](xref:Manifest.DeploymentOptions)|[0, 1]|Specifies options for registration and deployment of the package.|
-|&nbsp;&nbsp;[VersionHistory](xref:Manifest.VersionHistory)|[0, 1]|Contains an overview of the version history of this package.|
+|[Name](xref:Manifest.Name)||Specifies the name of the package.|
```

Comment:
don't remove spaces, these are there because the line above mentions a sequence and the indented items are part of that sequence